### ISSUE TITLE
 ng-pwa-tools Repository

### ISSUE THREAD

**Issue by mhadaily** [alxhub/io17#4]
Hi, 
the talk was very impressive. thanks for your hard work. However, I am to able to find ng-pwa-tools on Github, is there any public repository for that?
 Also, Like other people have said, your code in Master branch is not as similar to what you have done in the talk, would you please update to keep it as a reference. 

thanks

**Comment by ankemp** [comment]
https://www.npmjs.com/package/ng-pwa-tools

**Comment by mhadaily** [comment]
@ankemp I know that it's on NPM but I meant in Github.

**Comment by BorntraegerMarc** [comment]
Yeah the talk was great and I love the concept but I am very surprised that google would show something that is this early stage... I mean this is basically just an idea and nothing else...

**Comment by chaoranxie** [comment]
Totally agree with @BorntraegerMarc, i am not sure how mature the tools is given different issues posted on this repo. hopefully the author can chime in on the status of this project and when we should start using it. 

@mhadaily  if you just want to look at the source code, you can look into the `node_modules/ng-pwa-tools` folder

**Comment by mhadaily** [comment]
@chaoranxie of course I can, I was checking if it's public 📦 , and contribute to it.

**Comment by BorntraegerMarc** [comment]
If you end up here: Take a look at https://docs.google.com/document/d/1F0e0ROaZUnTFftmC0XovpREHWHjcXa4CggiFlmifjhw
Clarifies some questions...

**Comment by StephenFluin** [comment]
Here's the repo: https://github.com/alxhub/universal/tree/pwa-tools

**Comment by vladimir-barsuchenko** [comment]
@StephenFluin @alxhub 
Can't use `ng-pwa-tools` with Angular 5. Any ideas or replacements?

**Comment by Splaktar** [comment]
@levgaas you should use `@angular/service-worker` and the Angular CLI.

These [Service Worker Docs](https://angular.io/guide/service-worker-intro) and the follow on docs will take you through the process.

**Comment by vladimir-barsuchenko** [comment]
@Splaktar 
Do you know something about `ngu-firebase-push`? How to make it work for *v.5*

**Comment by Splaktar** [comment]
@levgaas no, sorry. I haven't used that library. I would post on Stackoverflow.
